.github/workflows/build-images.yml

```diff
@@ -0,0 +1,78 @@
+name: build-images
+
+on:
+  push:
+    branches:
+      - edge
+  release:
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  # packages: write
+
+jobs:
+  build-container-image:
+    name: Image Build ${{ matrix.name }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - name: Debian
+            file: Docker/Dockerfile
+            flavor: |
+              latest=auto
+              suffix=
+              onlatest=false
+          - name: Alpine
+            file: Docker/Dockerfile-Alpine
+            flavor: |
+              latest=auto
+              suffix=-alpine
+              onlatest=true
+    steps:
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v3
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
+
+    - name: Metadata
+      id: meta
+      uses: docker/metadata-action@v5
+      with:
+        flavor: ${{ matrix.flavor }}
+        images: |
+          docker.io/freshrss/freshrss
+      #     ghcr.io/${{ github.repository }}
+        tags: |
+          type=edge
+          type=semver,pattern={{version}}
+          type=semver,pattern={{major}}.{{minor}}
+          type=semver,pattern={{major}},enable=${{ !startsWith(github.ref, 'refs/tags/0.') }}
+        labels: |
+          org.opencontainers.image.description="A self-hosted RSS feed aggregator"
+          org.opencontainers.image.documentation="https://freshrss.github.io/FreshRSS/"
+          org.opencontainers.image.url="https://freshrss.org/"
```

Comment:
So what was the conclusion? Can't those static labels be removed, as they are already defined in the Dockerfile?

Reply:
This overwrite the metadata ones. And the metadata ones overwrite the statics ones.
So we benefit from the right versions label (and other labels)

Reply:
I am still confused: without these lines, what would `org.opencontainers.image.description` contain? Wouldn't it have what was defined in Dockefile?

Reply:
![grafik](https://github.com/FreshRSS/FreshRSS/assets/1573025/28afa6b7-8825-43f1-bd89-5369850d0e0b)

Reply:
when I dont set in manualy for the actions its taken from the github repo
![grafik](https://github.com/FreshRSS/FreshRSS/assets/1573025/0392d2d8-0872-4c79-90f1-b3b4b574ee85)

Reply:
The metadata action set it to `A free, self-hostable news aggregator… `I overwrite it to `A self-hosted RSS feed aggregator`because the static ones in the dockerfile would be overwritten by the metadata ones.

Reply:
Ok, thanks for the investigation 👍🏻